Docs/Userdocs/sm5_migration.md

```diff
@@ -0,0 +1,37 @@
+StepMania 5.1 to ITGmania Migration Guide
+=========================================
+
+This guide explains how to migrate from StepMania 5.1 to ITGmania without losing access to your settings and songs.
+
+The steps are as follows:
+
+1. Install ITGmania.
+2. Copy over songs, settings, profiles and other user data. (See detailed instructions below.)
+   If you don't intend on keeping StepMania 5.1 the data can be moved instead of copying.
+3. (skip for portable builds) ITGmania ships with Simply Love and the Club Fantastic song packs included. Duplicates of them can be removed from the user data directory.
+4. Uninstall StepMania 5.1 (optional).
+   If you choose to do so you can also uninstall the GrooveStats launcher as ITGmania has network functionality built in and doesn't need an external tool for that.
```

Comment:
Uninstalling the launcher is technically only currently supported for Simply Love. Waterfall and Digital Dance both still need the launcher for it them to work (as they haven't been updated yet, but they can still work with the launcher (once the launcher supports the itgmania name)

Reply:
good point, I didn't think of that! updated